### DISCUSSION TITLE
display only today's events until the endtime

### DISCUSSION THREAD

**Discussion by jonny-nX** [liedekef/events-made-easy#218] [Q&A]
Hi,
I would like to display only today's events via short code, up to the end of the respective event.

The 'scope--today' function does a good job, but only shows the event up to the start.

In the documentation I read that this function respects the start or end time.
Now how can I change this from the start time to the end time?
I've tried many things, but fail.

Thanks and best regards,
Johannes

**Comment by liedekef** [comment]
I'm not quite sure I understand, but by default show_ongoing=1 (as option to the eme_events shortcode) and that causes also ongoing events to be shown.

**Comment by jonny-nX** [comment]
Thanks for your answer!

I had tried that too.
But when today's event is over, the next one will be shown straight away - even if it's one day in the future.
That's exactly what I don't want...

I've managed to do it now. I used 'scope=show_ongoing' in the shortcode and an if query in the corresponding template as to whether the event is also today
[eme_if tag="#ESC_STARTDATE" value="#ESC_CURDATE"] ... [/eme_if]

**Reply by liedekef** [reply]
"show_ongoing" is not a valid option for scope, see https://www.e-dynamics.be/wordpress/category/documentation/6-shortcodes/eme_events/
Use "scope=today", which should be sufficient in your case.
